components/clarity-repl/src/analysis/ast_visitor.rs

```diff
@@ -177,9 +177,11 @@ pub trait ASTVisitor<'a> {
                                 }),
                         ),
                     };
-                } else if let Some(native_function) = NativeFunctions::lookup_by_name(function_name)
-                {
-                    use crate::clarity::functions::NativeFunctions::*;
+                } else if let Some(native_function) = NativeFunctions::lookup_by_name_at_version(
+                    function_name,
+                    &ClarityVersion::latest(), // FIXME(brice): this should probably be passed in
```

Comment:
Do we need to address this `TODO` now or later?

Reply:
It probably should be fixed to avoid a problem with user-defined functions in clarity 1 conflicting with native functions in clarity 2.